test/app-tap/module_api.c

```diff
@@ -1645,6 +1647,48 @@ test_key_def_merge(struct lua_State *L)
 	key_def_check_merge(a_25, lengthof(a_25), b_25, lengthof(b_25),
 			    exp_25, lengthof(exp_25));
 
+	/*
+	* Case 26: exclude_null=true + exclude_null=false.
+	*
+	* Interpretation: the same as for the case 3.
+	*/
```

Comment:
(Sorry if I comment too early and it is just draft. I got email regarding the changes and so glance on them.)

Multiline comments are formatted this way:

```
/*
 * foo
 * bar
 */
```

Linus regarding this: https://lkml.org/lkml/2016/7/8/625